CHANGELOG.md

```diff
@@ -1,22 +1,26 @@
 # Release History
 
-## 1.3.0 (TBD)
+## 1.3.0 (2023-03-28)
 
 ### New Features
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
-- Added API `Session.RuntimeConfig` for getting/setting/checking the mutability of any runtime configuration.
+- Added API `Session.conf` for getting/setting/checking the mutability of any runtime configuration.
 - Added support managing case sensitivity in `Row` results from `DataFrame.collect` using `case_sensitive` parameter.
+- Added indexer support for `snowflake.snowpark.types.StructType`.
+- Added a keyword argument `log_on_exception` to `Dataframe.collect` and `Dataframe.collect_no_wait` to optionally disable error logging for sql exceptions.
 
 ### Bug Fixes
 
 - Fixed a bug where a DataFrame set operation(`DataFrame.substract`, `DataFrame.union`, etc.) being called after another DataFrame set operation and `DataFrame.select` or `DataFrame.with_column` throws an exception.
+- Fixed a bug where chained sort statement is overwritten by SQL simplifier.
 
 ### Improvements
 
 - Simplified JOIN queries to use constant subquery aliases (SNOWPARK_LEFT, SNOWPARK_RIGHT) by default , users could disable this at runtime with `session.conf.set('use_constant_subquery_alias', False)` to use randomly generated alias names instead.
```

Comment:
Fix the comma splice and add backtiks around code words. Fixed verb tense in 2nd sentence.

"- Simplified JOIN queries to use constant subquery aliases (`SNOWPARK_LEFT`, `SNOWPARK_RIGHT`) by default. Users can disable this at runtime with `session.conf.set('use_constant_subquery_alias', False)` to use randomly generated alias names instead."